packages/zeplin-extension-style-kit/declarations/fontSrc.js

```diff
@@ -0,0 +1,47 @@
+import { STYLE_PROPS } from "../constants";
+
+const FORMATS = [{
+    format: "woff2",
+    ext: "woff2"
+}, {
+    format: "woff",
+    ext: "woff"
+}, {
+    format: "truetype",
+    ext: "ttf"
+}];
+
+class FontSrc {
+    constructor(fontFace) {
+        this.fontFace = fontFace;
+    }
+
+    static get DEFAULT_VALUE() {
+        return "";
+    }
+
+    get name() {
+        return STYLE_PROPS.FONT_SRC;
+    }
+
+    equals(other) {
+        return this.fontFace === other.fontFace;
+    }
+
+    hasDefaultValue() {
+        return this.fontFace === FontSrc.DEFAULT_VALUE;
+    }
+
+    getValue() {
+        if (this.hasDefaultValue()) {
+            return FontSrc.DEFAULT_VALUE;
+        }
+
+        const local = `local(${this.fontFace})`;
+        const external = FORMATS.map(({ ext, format }) => `url(/path/to/${this.fontFace}.${ext}) format("${format}")`).join(", ");
```

Comment:
I implemented it that way at first, but then tried to refactor it renaming FontSrc to Src, to make it sync with the proprety name, but then it didn't make sense to merge it since this also generates format next to the url. I also think that new lines and indentation would be nicer.